components/TeamPage/TeamMember.js

```diff
@@ -0,0 +1,30 @@
+import React, {useState} from 'react';
+import Constants from '../../global_constants/Constants';
+
+/**
+ * @returns {React.Component}
+ */
+const TeamMember = props => {
+
+    let select = false;
+    for (let i = 0; i < props.person.teams.length; i++) {
```

Comment:
I think you should use `.forEach` here instead. 

I don't know if I mentioned this before, but I actually got asked on an interview once when starting to use a `for int i = 0 i < etc` loop if I knew how to use `forEach` loops and I had to convince my interviewer that I knew how to use it effectively but was just choosing to use the other method during the interview.